travis/test_pylint

```diff
@@ -1,9 +1,47 @@
 #!/usr/bin/env python
 # -*- coding: utf-8 -*-
 
-"""
-Script to process pylint run and exit with result.
-"""
+
+'''This script runs `pylint` with plugins and special configuration files.
```

Comment:
https://www.python.org/dev/peps/pep-0008/#string-quotes: "For triple-quoted strings, always use double quote characters"